samples/.net/endpoints/batch/Clients/EndpointClient.cs

```diff
@@ -0,0 +1,79 @@
+﻿// <copyright file="EndpointClient.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// </copyright>
+
+using Azure.Core;
+using Azure.ResourceManager;
+using Azure.ResourceManager.MachineLearning;
+using BatchInferencingSamples;
+using System.Text;
+using System.Text.Json;
+
+namespace BatchInferencingSamples
+{
+    internal class BatchEndpointActions
+    {
+        public static async Task<BatchEndpointData> Get(Guid subscriptionId, string resourceGroup, string workspaceName, string batchEndpointName, TokenCredential tokenCredential)
+        {
+            ResourceIdentifier batchEndpointResourceId =
+                new ResourceIdentifier($"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/batchEndpoints/{batchEndpointName}");
+
+            ArmClient armClient = new ArmClient(tokenCredential);
+            var batchEndpoint = armClient.GetBatchEndpointResource(batchEndpointResourceId);
+            var data = await batchEndpoint.GetAsync().ConfigureAwait(false);
+            return data.Value.Data;
+        }
+
+        public static async Task Invoke(Uri scoringUri,Uri inputFolderUri, Uri outputFileUri, TokenCredential credentials)
+        {
+            var client = new HttpClient();
+
+            BatchScoringPayload payload = new BatchScoringPayload
+            {
+                Properties = new BatchScoringProperties
+                {
+                    InputData = new Dictionary<string, BatchScoringInput>
+                    {
+                        {
+                            "myInput",
+                            new BatchScoringInput
+                            {
+                                JobInputType = JobInputType.UriFolder,
+                                Uri = inputFolderUri,
+                            }
+                        },
```

Comment:
Could you use the following syntax, which helps reduce the number of curly braces and their indentations?
```
["myInput"] = new BatchScoringInput
{
    ...
}
```

Reply:
Alternatively creating a helper method would further reduce the noise: 
```
    BatchScoringPayload payload = new BatchScoringPayload
    {
        Properties = new BatchScoringProperties
        {
            InputData = CreateDictionary(
                "myInput",
                new BatchScoringInput
                {
                    JobInputType = JobInputType.UriFolder,
                    Uri = inputFolderUri,
                }),
            OutputData = CreateDictionary(
                "myOutput",
                new BatchScoringOutput
                {
                    JobOutputType = JobOutputType.UriFile,
                    Uri = outputFileUri
                })
        },
    };

    private static IDictionary<string, T> CreateDictionary<T>(string key, T value)
        => new Dictionary<string, T> { [key] = value };
        
```

Reply:
I opted for the first suggestion!